tuf/src/interchange/cjson/pretty.rs

```diff
@@ -1,15 +1,16 @@
 use serde::de::DeserializeOwned;
 use serde::ser::Serialize;
-use std::io::{Read, Write};
+use std::io::Read;
 
 use super::Json;
 use crate::interchange::DataInterchange;
 use crate::Result;
 
 /// Pretty JSON data interchange.
 ///
-/// This is identical to [Json] in all manners except for the `to_writer` method. Instead of
-/// writing the metadata in the canonical format, it instead pretty prints the metadata.
+/// This is identical to [Json] in all manners except for the `canonicalize` method. Instead of
```

Comment:
It's possible that this change breaks clients who are relying on to_writer. I'm not sure what rust-tuf's backwards compat guarantees are, but should we create another type instead?

Reply:
This is breaking compatibility, but we don't have any guarantees at the moment since we haven't released 0.3.0 yet. We shouldn't have trouble in fuchsia since we've vendored rust-tuf, and can upadte rust-tuf and any in-tree stuff in one CL.

 I'm mainly trying to do a lot of the cleanup like this so I can cut a 0.3.0 release.